src/styles/_card.scss

```diff
@@ -0,0 +1,102 @@
+@import "./variables.scss";
+
+p,
+a {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
It's not related to the card. So move it outside the card.scss file. You can create a separate file for resetting